tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -83,58 +91,74 @@ def test_gets_domains_with_cc_and_bcc(self):
         self.assertSetEqual(domains, {'bar.com', 'com'})
 
 
+class ResizeImage(TestCase):
+    def test_change_image_size(self):
+        input_base64 = _given_test_image_base64(size='large')
+        output_base64 = email_parser._change_image_size(input_base64)
+        self.assertNotEqual(input_base64, output_base64, self.getErrorMessage())
+
+    def test_change_image_size_when_already_small(self):
+        input_base64 = _given_test_image_base64(size='small')
+        output_base64 = email_parser._change_image_size(input_base64)
+        self.assertEqual(input_base64, output_base64, self.getErrorMessage())
```

Comment:
You can have an even stronger check here using `assertIs` to ensure that it's the exact same string reference instead of just having equal strings.

Same below.